src/frontend/Sider/Behandling/Vilkårvurdering/EndreVilkår.tsx

```diff
@@ -207,6 +209,7 @@ export const EndreVilkår: FC<EndreVilkårProps> = (props) => {
                 size="small"
                 value={fom}
                 feil={feilmeldinger.fom}
+                readOnly={props.felterSomKanRedigeres == 'ALLE'}
```

Comment:
Fom er redigerbart når jeg satt revurderFra oktober 
<img width="423" alt="image" src="https://github.com/user-attachments/assets/e8a89f71-31f1-4c12-8ec4-d3743fb9d519">

```suggestion
                readOnly={props.felterSomKanRedigeres != 'ALLE'}
```

Reply:
Gjelder de andre `== ALLE` i denna fila og

Reply:
Damn. I fix!